charts/argo-rollouts/templates/controller/role.yaml

```diff
@@ -68,6 +68,7 @@ rules:
   - get
   - list
   - watch
+  - update
```

Comment:
We also have [clusterrole](https://github.com/argoproj/argo-helm/blob/main/charts/argo-rollouts/templates/controller/clusterrole.yaml#L59-L69), so it should be fixed at the same time. 🤔 

*Though upstream is not released yet.